enterprise/api.py

```diff
@@ -7,10 +7,19 @@
 from enterprise.models import EnterpriseCustomerBrandingConfiguration
 
 
-def get_enterprise_branding_info(provider_id=None):
+def get_enterprise_branding_info(provider_id=None, ec_uuid=None):
```

Comment:
I like this idea -- it's good to have very specific functions representing these pieces of system behavior with their own test coverage.